### DISCUSSION TITLE
能不能增加查看浏览记录和自己发的帖子的功能

### DISCUSSION THREAD

**Discussion by Lin559** [share121/inter-knot#2661] [其他]
rt
昨晚睡觉前想到一个绝妙的回复，但今早我就找不到想回复的帖子了。<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >
同时早上进邮箱一看40+条留言不仅混乱而且无法直观了解到帖子的情况。<img src="https://img-static.mihoyo.com/communityweb/upload/9273d3a47c5c35fbc746338625868d83.gif" alt="苦酒入喉" width="75" >

**Comment by Lin559** [comment]
欸，第一个表情包变成封面了<img src="https://img-static.mihoyo.com/communityweb/upload/dce2b6d619041e9f7ad133c804fb1a0d.png" alt="妮可_震惊" width="75" >

**Comment by lllliufeng** [comment]
可以看[这个](https://docs.github.com/zh/search-github/searching-on-github/searching-discussions)
按评论者搜索和按作者搜索可以在讨论页面实现这些功能(～o￣▽￣)～o

**Reply by lllliufeng** [reply]
![image](https://github.com/user-attachments/assets/93face81-2ded-4d30-b88a-c6a016a4324f)
![image](https://github.com/user-attachments/assets/cb130aa2-9b75-462b-8399-41f0d23b3cf1)

**Reply by Lin559** [reply]
感谢解答，已经用上了

**Comment by HKLHaoBin** [comment]
### 呐~这就是你的浏览记录。

![Clip_2024-08-18_21-02-01](https://github.com/user-attachments/assets/5c2a815d-149c-4273-858e-3f44657f45c5)

<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by Lin559** [reply]
感谢大佬解答

**Comment by yansteve454** [comment]
确实

**Comment by LeiMoo** [comment]
邮件你设置一下归档或者说收信规则，下面是qq邮箱的设置
![image](https://github.com/user-attachments/assets/269eb6c6-7d68-4e1c-8c34-2eade73fae27)

![image](https://github.com/user-attachments/assets/cae85657-3012-4b69-b156-713fb4bd7321)

其他邮箱也有相似功能。
